listenbrainz/webserver/static/js/src/listens/ListenCard.tsx

```diff
@@ -46,6 +46,42 @@ export default class ListenCard extends React.Component<
   ListenCardProps,
   ListenCardState
 > {
+  static preciseTimestamp = (listen: Listen): string => {
+    const listened_at = listen.listened_at_iso || listen.listened_at * 1000;
+    const listenDate: Date = new Date(listened_at);
+    const msDifference = new Date().getTime() - listenDate.getTime();
+    if (
+      // over one year old : show with year
+      msDifference / (1000 * 3600 * 24 * 365) >
```

Comment:
I'm just opening a suggestion here - should it be a listen over 365 days old, or should it be any listen from last year? e.g. in Feb 2021, a listen from Dec 2020 will include the year

Reply:
I think if I'm in Feb 2021 and I see a date in March, I'll assume it's that of the past year. 
I think this assumption works well and we end up with a slightly less formal looking date.